opentreemap/treemap/instance.py

```diff
@@ -371,6 +371,17 @@ def get_udfc_search_fields(self, user):
 
         return data
 
+    def unlocked_udf_models(self):
```

Comment:
What do you think about ``editable_udf_models`` instead of ``unlocked_udf_models``. "Lock" is not a verb we have used anywhere else in the python code.